opencti-platform/opencti-graphql/src/database/file-storage-helper.ts

```diff
@@ -103,7 +123,8 @@ export const deleteAllBucketContent = async (context: AuthContext, user: AuthUse
     const ids = [];
     for (let fileI = 0; fileI < allFiles.length; fileI += 1) {
       const currentFile = allFiles[fileI];
-      if (currentFile) {
+      logApp.info(`[FILE STORAGE] preparing for delete ${currentFile}`);
+      if (currentFile && currentFile.id.length > 0) {
```

Comment:
```suggestion
      if (currentFile?.id) {
```
seems sufficient if you just need to check that the id is a non-empty string